cypress/e2e/crypto/utils.ts

```diff
@@ -59,7 +59,6 @@ export function handleSasVerification(verifier: Verifier): Promise<EmojiMapping[
 
         // @ts-ignore as above, avoiding reference to VerifierEvent
         verifier.on("show_sas", onShowSas);
-        verifier.verify();
```

Comment:
with `startVerification`, the `verify` is implicit, so this is no longer needed.